### PR TITLE
Adds symlog plotting mode and makes that the default for plots

### LINKED ISSUE #12: log scale enhancements
Log scale should be "symlog" to support the negative numbers in the normalized plots. Also, log scale should disable clipping but preserve reasonable y-axis ticks.

### PR DESCRIPTION
**Pull request by kelloggm**
Fixes #12. Symlog plotting removes the need for clipping at the top and bottom.

### CODE REVIEW

**Comment by kelloggm** [comment]
@mwillsey

File: crossbot/commands/plot.py [hunk 1]
```diff
@@ -55,11 +55,16 @@ def init(client):
...
         dest   = 'scale',
         const  = 'log')
-    scales.add_argument (
+    scales.add_argument(
         '--linear',
         action = 'store_const',
         dest   = 'scale',
         const  = 'linear')
+    scales.add_argument(
+        '--symlog',
```
Comment (mwillsey) [inline crossbot/commands/plot.py]: `symlog` should replace the `log` option, but under the name `--log`. Symmetric log is always better for this case.

File: crossbot/commands/plot.py [hunk 2]
```diff
@@ -105,7 +110,7 @@ def plot(client, request):
     `smoothing` is between 0 (no smoothing) and 1 exclusive. .6 default
     You can provide either `num_days` or `start_date` and `end_date`.
     `plot` plots the last 5 days by default.
-    The scale can be `log` or `linear`.'''
+    The scale can be `log`, `symlog`, or `linear`.'''
```
Comment (mwillsey) [inline crossbot/commands/plot.py]: `log` shouldn't be an option.

File: crossbot/commands/plot.py [hunk 3]
```diff
@@ -55,11 +55,16 @@ def init(client):
         action = 'store_const',
         dest   = 'scale',
         const  = 'log')
```
Comment (mwillsey) [inline crossbot/commands/plot.py]: just have this store `symlog`

File: crossbot/commands/plot.py [hunk 4]
```diff
@@ -127,7 +132,7 @@ def plot(client, request):
         request.reply('smooth should be between 0 and 0.95', direct=True)
 
     if args.scale is None:
-        args.scale = 'linear' if args.plot_type == 'normalized' else 'log'
+        args.scale = 'symlog'
```
Comment (mwillsey) [inline crossbot/commands/plot.py]: This if isn't needed anymore. Use the `set_defaults` above.

File: crossbot/commands/plot.py [hunk 5]
```diff
@@ -241,7 +249,7 @@ def fmt_min(sec, pos):
                 color = 'red' if args.focus == name else '#0F0F0F0F'
             ax.plot_date(mdates.date2num(dates), seconds, next(markers), label=name, color=color)
 
-        if args.scale == 'log':
+        if args.scale == 'log' or args.scale == 'symlog':
```
Comment (mwillsey) [inline crossbot/commands/plot.py]: no more `log`

**Comment by kelloggm** [comment]
@mwillsey I've removed the "log" option and just replaced it with symlog

**Comment by kelloggm** [comment]
Fixed the last thing you pointed out, this should be mergable.

**Comment by mwillsey** [comment]
So for some reason the default `crossbot plot` doesn't have y-axis ticks. I tried to play around a little bit by manually setting the [ticker](https://matplotlib.org/api/ticker_api.html), but it was putting linear ticks on a log graph!

Tick labels are important though, so I probably wont merge until it works.

**Comment by mwillsey** [comment]
Manually merged in c6b6ca3d46ac6891c4112aca5a47de2a67af83f0.

**Pull request closed without merging**